recipes/perl-compress-bgzf/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "perl-compress-bgzf" %}
+{% set version = "0.006" %}
+{% set sha256 = "fb9716ba560a19d2cf29f385f5bf76d9c362fb71ba9df1ada244450230b1da67" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/V/VO/VOLKENING/Compress-BGZF-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  skip: true  # [win]
+  script:
+    - perl Build.PL 
+    - perl ./Build
+    - perl ./Build test
+    - perl ./Build install --installdirs vendor --create_packlist 0
+
+requirements:
+  build:
+
+  host:
+    - perl
+    - perl-module-build
+
+  run:
+    - perl
```

I suggest we tidy this up:

```suggestion
  host:
    - perl
    - perl-module-build
  run:
    - perl
```

---

Thanks, just pushed a commit with the suggested changes.